apps/website/src/server/notifications.ts

```diff
@@ -209,15 +210,12 @@ async function createDiscordNotifications({
 
   const tasks = [];
   for (const webhookUrl of webhookUrls) {
-    const relativeNotificationUrl = `/notifications/${id}`;
-    const link =
-      linkUrl ||
-      new URL(relativeNotificationUrl, env.NEXT_PUBLIC_BASE_URL).toString();
```

Comment:
We seem to have lost this logic here for defaulting to the notification URL if one isn't set?